pkg/operation/botanist/secrets.go

```diff
@@ -441,6 +467,21 @@ func (b *Botanist) SyncShootCredentialsToGarden(ctx context.Context) error {
 		},
 	}
 
+	// ssh-keypair.old secret will be synced to the Garden cluster if it is present in shoot namespace in seed.
+	oldSecret := &corev1.Secret{}
+	if err := b.K8sSeedClient.Client().Get(ctx, kutil.Key(b.Shoot.SeedNamespace, v1beta1constants.SecretNameOldSSHKeyPair), oldSecret); err != nil {
+		if !apierrors.IsNotFound(err) {
+			return err
+		}
+	} else {
+		b.StoreSecret(v1beta1constants.SecretNameOldSSHKeyPair, oldSecret)
```

Comment:
This line can be removed now, right? You already store it here: https://github.com/gardener/gardener/pull/5388/files#diff-5a59ce10d2c8c2ae85d9458b6e256e7e957d3b21b160380bb121fddb4a98befcR235